### DISCUSSION TITLE
Multimeasure trills still not playing correctly

### DISCUSSION THREAD

**Discussion by toysmith** [musescore/MuseScore#24080] [General]
After [leaving a comment in the MuseScore forum](https://musescore.org/en/node/323436), someone noted the issue tracker has been discontinued and that I should post this on GitHub.  I hope this is the right forum for a bug report; please let me know if/where I should re-post if not.

Multimeasure trills still do not play back correctly - they break after the first measure or so.  I'm trying to attach a demo file but don't seem to have permissions to upload a MuseScore file?  The issue was originally reported in MuseScore 3 an is still present in 4.3.2.

I'm happy to help diagnose further!

**Comment by MarcSabatella** [comment]
The correct place to report issues is on the "issues" page, not "discussions" - see https://github.com/musescore/MuseScore/issues.  And the instructions there will help in terms of showing you how to ZIP and attach a score, also in advising you to first search for existing issues.
